src/js/rules/camel-case-func.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const shims = require('../util/shims');
+
+module.exports = {
+    create(context) {
+        const report = (astNode) => {
+            astNode.children.forEach(element => {
```

Comment:
Instead of iterating over every child of the AST here, couldn't we just look at the node at position 1 like we were before? If not, could we at least exit early so that we don't continue iterating?

Reply:
Hmm this was added before I implemented some of the newer shims. Updating to use `findFirst` which exits the loop early.